controllers/pattern_controller.go

```diff
@@ -505,3 +545,105 @@ func (r *PatternReconciler) actionPerformed(p *api.Pattern, reason string, err e
 	}
 	return r.onReconcileErrorWithRequeue(p, reason, err, nil)
 }
+
+// MBP-184 Implementation
+// applyPatternAppDetails retrieves the status for the Pattern applications created in ArgoCD.
+// Once the Application Status is retrieve it appends the information
+// to the array and the CR is updated.
+func (r *PatternReconciler) applyPatternAppDetails(client argoclient.Interface, input *api.Pattern) (*api.Pattern, error) {
+	output := input.DeepCopy()
+	var applicationInfo api.PatternApplicationInfo
+
+	// Buid the namespace from the Pattern ObjectMeta.Name and Spec.ClusterGroupName
+	ns := output.ObjectMeta.Name + "-" + output.Spec.ClusterGroupName
+	// oc get Applications -n [pattern-name]-hub
+	// E.G. oc get Applications -n multicloud-gitops-hub
+	applications, err := client.ArgoprojV1alpha1().Applications(ns).List(context.Background(), metav1.ListOptions{})
+
+	// Return error
+	if err != nil {
+		return output, err
+	}
+
+	// Reset the array since we are going to replace it with the new application status
+	if len(output.Status.Applications) > 0 {
+		output.Status.Applications = nil
+	}
+
+	// Loop through the Pattern Applications and append the details to the Applications array
+	for _, app := range applications.Items {
+		//fmt.Println(app.Name, " ==> [", app.Status.Health.Status, "] == [", app.Status.Sync.Status, "]")
+		// Add Application information to ApplicationInfo struct
+		applicationInfo.Name = app.Name
+		applicationInfo.Namespace = app.Spec.Destination.Namespace
```

Comment:
Ah, I was actually thinking 
```suggestion
		applicationInfo.Namespace = app.Namespace
```

so that someone looking for additional detail would know where to find the application